avm/src/daemon/daemon.rs

```diff
@@ -232,13 +233,13 @@ async fn generate_token() -> String {
   post_v1("localDaemonToken", body).await
 }
 
-async fn set_local_daemon_props() -> () {
+async fn set_local_daemon_props(agz_b64: Option<String>) -> () {
   let files_b64 = get_files_b64().await;
   create_app_tar(files_b64.get(&"app.tar.gz".to_string()));
   DAEMON_PROPS
     .set(DaemonProperties {
       clusterId: "daemon-local-cluster".to_string(),
-      agzB64: get_base_agz_b64(),
+      agzB64: agz_b64.unwrap(),
```

Comment:
if we unconditionally `agz_b64.unwrap()` then it should just be a `String`, not `Option<String>`

Reply:
I'm checking it here: https://github.com/alantech/alan/pull/504/files#diff-583f2380a4d780cb04bd93e3492a6f2828052a50e23460f3a2c8d89cf3ade79fR277-R283

Reply:
sure, but that's not guaranteed at every call to `set_local_daemon_props`, so somebody who's contributing and calls that fn might pass in a `None` and then everything fails

Reply:
Oh thats true, got it